README.md

```diff
@@ -112,6 +114,8 @@ To install this library via [npm](https://www.npmjs.org) **version 3**, use the
 npm install @reactivex/rxjs
 ```
 
+This will include all builds for CJS/ES6/AMD/TypeScript.
```

Comment:
Well, I think this PR's crux is to resolve those question :) if you're using cjs only, there is no difference between two package. @reactivex scoped package is intended for someone who wants to have full builds other than cjs such as global (and potentially es6 once it's sorted out). If you prefer shorter package, then it's good to go. Is this answers for you?